samples/fuel_gauge/max17048/README.rst

```diff
@@ -0,0 +1,51 @@
+.. _MAX17048_sample:
+
+MAX17048 Li-Ion battery fuel gauge
+###################################
+
+Overview
+********
+
+This sample shows how to use the Zephyr :ref:`sensor_api` API driver for the `MAX17048` sensor.
+
+.. _MAX17048: https://www.maximintegrated.com/en/products/power/battery-management/MAX17048.html
+
+The sample periodically reads battery percentage and power status
+
+Building and Running
+********************
+
+The sample can be configured to support MAX17048 sensors connected via either I2C. It only needs
+an I2C pin configuration
+
+Features
+********
+By using this sensor you can get the following information:
```

Comment:
Fuel guage is technically a sensor but might be a good idea to just change `sensor` to `fuel gauge` here in case someone confuses it with the `sensor` API.